### PR TITLE
More CKAN 2.9 Compatibility

### PR DESCRIPTION
**Pull request by JVickery-TBS**
Controller fix; Template updates for View routes; CSS fixes for user avatar;

### CODE REVIEW

File: ckanext/canada/public/css/canada1.css [hunk 1]
```diff
@@ -359,4 +359,24 @@ a#edit-comment-body-0-format-help-about, button#edit-submit,button#edit-actions-
...
+    }
+}
+.user-image-wrapper img.user-image {
+    position: absolute;
+    max-height: 100%;
+    max-width: 100%;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
```
Comment (wardi) [inline ckanext/canada/public/css/canada1.css:381]: would you add some comments to indicate what these are intended to fix?

Comment (JVickery-TBS) [reply ckanext/canada/public/css/canada1.css]: @wardi this had to do with the user avatars. It has been removed now.

File: ckanext/canada/templates/public/user/read_base.html [hunk 2]
```diff
@@ -10,7 +10,9 @@
     <div class="panel-body">
       {% if h.gravatar_show() %}
         {% block user_image %}
-          {{ super () }}
+          <div class="user-image-wrapper">
+            <div class="image">{{ h.user_image(user.id, size=230) }}</div>
+          </div>
```
[the remark this answers is not in the document]
Comment (JVickery-TBS) [reply ckanext/canada/templates/public/user/read_base.html]: @wardi everything to do with the user images/avatars has been removed from our extension and templates. And the Public plugin class now sets the default avatar to disabled.

File: ckanext/canada/view.py [hunk 3]
```diff
@@ -92,12 +108,39 @@ def post(self, package_type, id):
...
+        phoneno=params.get('phoneno', '')
+        dept=params.get('department', '')
+        response = super(CanadaUserRegisterView, self).post()
+        if hasattr(response, 'status_code'):
+            if response.status_code == 200 or response.status_code == 302:
+                # redirected after successful user create
+                import ckan.lib.mailer
+                # checks if there is a custom function "notify_ckan_user_create" in the mailer (added by ckanext-gcnotify)
+                getattr(
+                  ckan.lib.mailer,
```
Comment (wardi) [inline ckanext/canada/view.py:126]: let's use 4-space indents throughout python code in the repo

**Comment by wardi** [comment]
looks good. love to see unused code being removed.

File: ckanext/canada/view.py (hunk 3 above)
Comment (JVickery-TBS) [reply ckanext/canada/view.py]: @wardi oops, just copied that code over from the Controller. Is fixed now.

Comment (wardi) [reply ckanext/canada/view.py]: yep, might as well clean it up now.

**Pull request merged**